### ISSUE TITLE
Warning: Calling fails_with :gcc_4_0 is deprecated! There is no replacement.

### ISSUE THREAD

**Issue by dylanjm** [llvm-hs/homebrew-llvm#9]
Trying to update packages using homebrew on macOS and I keep getting this warning message:

```
Warning: Calling fails_with :gcc_4_0 is deprecated! There is no replacement.
Please report this to the llvm-hs/llvm tap:
  /usr/local/Homebrew/Library/Taps/llvm-hs/homebrew-llvm/Formula/llvm-5.0.rb:96
```

My current `gcc --version` is

```
gcc --version
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
Apple LLVM version 10.0.0 (clang-1000.11.45.5)
Target: x86_64-apple-darwin18.2.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

**Comment by tmcdonell** [comment]
Thanks for letting me know. This should be fixed now.
